test/hotspot/jtreg/serviceability/jdwp/DebuggeeLauncher.java

```diff
@@ -129,16 +97,7 @@ public int getJdwpPort() {
     }
 
     @Override
-    public void onStringRead(StreamHandler handler, String line) {
-        if (handler.equals(errorHandler)) {
-            terminateDebuggee();
-            listener.onDebuggeeError(line);
-        } else {
-            processDebuggeeOutput(line);
-        }
-    }
-
-    private void processDebuggeeOutput(String line) {
+    public void onStringRead(String line) {
```

Comment:
Is there a reason not to also echo the stdout output? Seems it would be useful if the test ever failed. Doing so probably means adding back in the StreamHandler argument.

Reply:
I don't like approach to compare handlers. Better to just add logging into these listeners.

Reply:
updated

Reply:
stdout logging could be improved if needed when test fails.